Sources/FluentUI_iOS/Components/Pill Button Bar/PillButtonTokenSet.swift

```diff
@@ -182,23 +244,35 @@ public class PillButtonTokenSet: ControlTokenSet<PillButtonToken> {
 }
 
 extension PillButtonTokenSet {
-    /// The distance of the content from the bottom of the `PillButton`.
-    static let bottomInset: CGFloat = GlobalTokens.spacing(.size60)
+    /// The corner radius of the `PillButton`.
+    static let cornerRadius: CGFloat = GlobalTokens.spacing(.size160)
```

Comment:
wouldn't the corner radius always be height/2?

Reply:
Or use `GlobalTokens.corner(.radiusCircular)`